docs/source/operations/howto/connect-organizations.rst

```diff
@@ -0,0 +1,98 @@
+*************************************
+How to link multiple Substra backends
+*************************************
+
+This guide shows you how to enable communications between two Substra backends to make them exchange assets.
+This can be achieved either at deployment time or when the bakcend is already deployed.
+
+Prerequisites
+=============
+
+You will need `Helm <https://helm.sh>`_.
+
+Updating your Helm values
+=========================
+
+For the purpose of this guide we will use a setup with three Substra backends as shown in :ref:`Figure 1 <figure-1>`.
+
+.. _figure-1:
+.. mermaid:: diagrams/howto-link-backends-channels.mmd
+
+**Figure 1.** A channel with three nodes
```

Comment:
What is the role of MyOrg3? Is it because you plan to update the doc to include a third backend?

Reply:
I was hesitant on this but when writing the doc it felt confusing with 3 orgs. It was to add an org that is not included in the asset exchange loop.

Reply:
So from what I understand, MyOrg3 would be in the same channel as MyOrg1 and MyOrg2, but the user wouldn't be able to train a model on data from (MyOrg3 plus (MyOrg1 and/or MyOrg2)) that's it ? 

For simplicity of understanding, I would suggest to either consider a scenario where the 3 organizations can communicate, or to keep only two if you think that it feels too confusing when writing the doc. 

But happy to get your feedback or the one from other reviewers as well!